app/liquid_tags/stackexchange_tag.rb

```diff
@@ -1,24 +1,28 @@
 class StackexchangeTag < LiquidTagBase
   PARTIAL = "liquids/stackexchange".freeze
-  # update API version here and in stackexchange_tag_spec when a new version is out
-  API_URL = "https://api.stackexchange.com/2.2/".freeze
+  REGISTRY_REGEXP = %r{https://(?<subdomain>\w*)(?:\.)?(?:stackexchange.com|stackoverflow.com)/(?<post_type>q|a|questions)/(?<id>\d{1,20})(?:/)?(?:[\w\-]*)}
```

Comment:
As written the regular expression would allow `fakestackexchange.com` as a valid URL.  This is because the `(?:\.)?` is optional.

I believe we want `(?<subdomain>\w*)(?:\.)?`: to be `(?:(?<subdomain>\w+)\.)?`

Reply:
Great catch, thank you!